CyFile-Android/app/src/main/res/layout/activity_display_note.xml

```diff
@@ -19,6 +19,19 @@
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/noteText" />
 
+    <Button
+        android:id="@+id/BTN_DEL"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
```

Comment:
Instead of using hardcoded values for Margin please use "dimen/default_margin_dp" property for it.